packages/create-sitecore-jss/src/templates/angular/package.json

```diff
@@ -1,119 +1,119 @@
-    "karma-jasmine-html-reporter": "~1.5.4",
-    "move-cli": "^2.0.0",
-    "npm-run-all2": "^7.0.1",
-    
-    "ts-node": "~10.9.2",
-    "typescript": "~5.4.0"
-  },
-  "overrides": {
-    "request": {
-      "form-data": "^2.5.5"
-    }
-  }
-}
+{
+  "name": "<%- appName %>",
+  "version": "0.1.0",
+  "description": "Application utilizing Sitecore JavaScript Services and Angular (angular-cli).",
+  "config": {
+    "appName": "<%- appName %>",
+    "rootPlaceholders": [
+      "<%- helper.getAppPrefix(appPrefix, appName) %>jss-main"
+    ],
+    "sitecoreDistPath": "/dist/<%- appName %>",
+    "sitecoreConfigPath": "/App_Config/Include/zzz",
+    "buildArtifactsPath": "./dist",
+    "graphQLEndpointPath": "/sitecore/api/graph/edge",
+    "language": "en"
+  },
+  "scripts": {
+    "ng": "ng",
+    "lint": "ng lint <%- appName %> --fix",
+    "jss": "jss",
+    "start:connected": "npm-run-all --serial bootstrap start:angular start:watch-components",
+    "build": "npm-run-all --serial bootstrap build:client build:server",
+    "scaffold": "ng generate @sitecore-jss/sitecore-jss-angular-schematics:jss-component --no-manifest",
+    "start:angular": "ng serve -o",
+    "start:watch-components": "ts-node --project src/tsconfig.webpack-server.json scripts/generate-component-factory/index.ts --watch",
+    "build:client": "cross-env-shell ng build --configuration=production --base-href $npm_package_config_sitecoreDistPath/browser/ --output-path=$npm_package_config_buildArtifactsPath/browser/",
+    "build:server": "cross-env-shell ng run <%- appName %>:server:production --output-path=$npm_package_config_buildArtifactsPath",
+    "postbuild:server": "move-cli ./dist/main.js ./dist/server.bundle.js",
+    "bootstrap": "ts-node --project src/tsconfig.webpack-server.json scripts/bootstrap.ts",
+    "graphql:update": "ts-node --project src/tsconfig.webpack-server.json ./scripts/update-graphql-fragment-data.ts",
+    "install-pre-push-hook": "ts-node --project src/tsconfig.webpack-server.json ./scripts/install-pre-push-hook.ts"
+  },
+  "private": true,
+  "author": {
+    "name": "Sitecore Corporation",
+    "url": "https://jss.sitecore.com"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/sitecore/jss.git"
+  },
+  "bugs": {
+    "url": "https://github.com/sitecore/jss/issues"
+  },
+  "homepage": "https://jss.sitecore.com",
+  "license": "Apache-2.0",
+  "dependencies": {
+    "@angular/animations": "~20.3.10",
+    "@angular/common": "~20.3.10",
+    "@angular/compiler": "~20.3.10",
+    "@angular/core": "~20.3.10",
+    "@angular/forms": "~20.3.10",
+    "@angular/platform-browser": "~20.3.10",
+    "@angular/platform-browser-dynamic": "~20.3.10",
+    "@angular/platform-server": "~20.3.10",
+    "@angular/router": "~20.3.10",
+    "@apollo/client": "^3.12.9",
+    "@ngx-translate/core": "~15.0.0",
+    "@ngx-translate/http-loader": "~8.0.0",
+    "@sitecore-cloudsdk/events": "^0.5.2",
```

Comment:
This package is a part of angular-xmcloud template. It should be removed from base